src/main/webapp/app/exam/manage/exercise-groups/programming-exercise-cell/programming-exercise-group-cell.component.html

```diff
@@ -84,14 +84,19 @@
     @if (displayEditorModus) {
         <div>
             <div class="d-flex justify-content-between">
-                <span jhiTranslate="'artemisApp.programmingExercise.offlineIde'">: {{ programmingExercise.allowOfflineIde || false }}</span>
+                <span [jhiTranslate]="'artemisApp.programmingExercise.offlineIde'"></span>:
+                <span [jhiTranslate]="programmingExercise.allowOfflineIde ? 'artemisApp.exercise.yes' : 'artemisApp.exercise.no'"></span>
```

Comment:
In my opinion, this would look nicer with less spacing:

![image](https://github.com/user-attachments/assets/c36ef1de-526e-4e36-8e05-3ecd5bea98f7)

We could remove the space between the colon and the label using the class `colon-suffix`:

```suggestion
                <span class="colon-suffix" [jhiTranslate]="'artemisApp.programmingExercise.offlineIde'"></span>
                <span [jhiTranslate]="programmingExercise.allowOfflineIde ? 'artemisApp.exercise.yes' : 'artemisApp.exercise.no'"></span>
```

![image](https://github.com/user-attachments/assets/98c88632-fc1d-4245-aa0c-1eaa59152901)


(optional) But I wonder if we really need to have this much space here in the first place. Maybe you could also remove the `justify-content-between` from the parent element and replace it with `gap-1`

(so `<div class="d-flex gap-1" >` for the parent div of these span elements)

![image](https://github.com/user-attachments/assets/752cf3b5-b1b2-4952-a650-c7f905e71e9e)